src/Node.tsx

```
@@ -1,56 +1,98 @@
+
+  useEffect(() => {
+        setFocus(focus)
+  }, [_focus])
+
+  const onWordTyped = (e: ChangeEvent<HTMLInputElement>) => {
+      setTypedWord(e.target.value);
+      console.log(e.target.value)
+      const filtered = getWords().filter(({key}: {key: string}) => key != name)
+      setWords([...filtered, {key: name, typedWord: e.target.value}])
+      setWordObject(wordObject?.map((obj, i)=> {
+          let new_color = e.target.value[i] == obj.letter ? "green" : obj.color
```

ESLint

[error] 'new_color' is never reassigned. Use 'const' instead.

(prefer-const)